docs/_posts/2018-11-19-android.md

```diff
@@ -672,6 +672,38 @@ marathon {
 
 {% endtab %} {% endtabs %}
 
+## Multiple adb servers
+Default configuration of marathon assumes that adb server is started locally and is available at `127.0.0.1:5037`. In some cases it may be
+desirable to connect multiple adb servers instead of connection devices to one adb servers. An example of this is distributed execution of
```

Comment:
I would be nice to have “ In order to expose the adb server it should be binded to public interface, e.g. adb nodaemon server -a -P 5037” in docs

Reply:
Sure thing

Reply:
@tagantroy Done